MicrowaveGame/Assets/Resources/Scripts/SceneFader.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.SceneManagement;
+
+public class SceneFader : MonoBehaviour
```

Comment:
Behaviours should have `Behaviour` suffixed on the file name and class name (i.e. `SceneFaderBehaviour` and `SceneFaderBehaviour.cs`).